cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -151,11 +151,11 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
                 "times. For example: --label distribution=external"
     ).associate()
 
-    private val activatedPackageManagers by option(
+    private val enabledPackageManagers by option(
         "--package-managers", "-m",
-        help = "The comma-separated package managers to activate, any of ${allPackageManagersByName.keys}. Note that " +
-                "deactivation overrides activation. If set, package manager activation settings from configuration " +
-                "files are ignored."
+        help = "The comma-separated package managers to enable, any of ${allPackageManagersByName.keys}. Note that " +
+                "disabling overrides enabling. If set, package manager enabling settings from configuration files " +
+                "are ignored."
```

Comment:
While at it you could rephrase the sentence to be a more specific, for example: "If set, the 'enabledPackageManagers' property from configuration files is ignored." Same for the option below.

Reply:
I'm usually not a fan of hard-coding other option names from an option's help text as such references are hard to maintain. But I'll do it here nonetheless as the other options are being referred to in the deprecation messages already anyway.